intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -81,6 +81,30 @@ outputs = model.generate(inputs, streamer=streamer, max_new_tokens=300)
 
 ```
 
+Augument description of generate:
+| Augument             | Description                                                              |
+| --------------       | -----------------------------------------------------------------------  |
+| inputs               | input ids after tokenizer                                                |
```

Comment:
better to indicate the datatype type like
```suggestion
| inputs             | uint32_t | input ids after tokenizer                                                |
```

Reply:
uint32_t is not used by python...

Reply:
then ```int``` should be enough